.github/workflows/prerelease.yml

```diff
@@ -124,6 +124,40 @@ jobs:
         author_name: Failure in linting provider
         fields: repo,commit,author,action
         status: ${{ job.status }}
+  lint_sdk:
+    container: golangci/golangci-lint:latest
+    name: lint-sdk
+    needs: build_sdk
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+    - name: Checkout Scripts Repo
+      uses: actions/checkout@v2
+      with:
+        path: ci-scripts
+        repository: pulumi/scripts
+    - name: Unshallow clone for tags
+      run: git fetch --prune --unshallow
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.goversion }}
+    - name: Install pulumictl
+      uses: jaxxstorm/action-install-gh-release@release/v1-alpha
+      with:
+        repo: pulumi/pulumictl
+    - name: Install Pulumi CLI
+      uses: pulumi/action-install-pulumi-cli@releases/v1
+    - name: Run golangci
+      run: cd sdk/go/azuredevops && golangci-lint run -c ../../../.golangci.yml
```

Comment:
This creates an error if there are no go files
![image](https://user-images.githubusercontent.com/14177833/86141378-49ba0080-baf2-11ea-9de3-4d234c7637b5.png)